src/app.tsx

```diff
@@ -65,37 +65,22 @@ export const layout: RunTimeLayoutConfig = ({ initialState }) => {
   };
 };
 
-const codeMessage = {
-  200: '服务器成功返回请求的数据。',
-  201: '新建或修改数据成功。',
-  202: '一个请求已经进入后台排队（异步任务）。',
-  204: '删除数据成功。',
-  400: '发出的请求有错误，服务器没有进行新建或修改数据的操作。',
-  401: '用户没有权限（令牌、用户名、密码错误）。',
-  403: '用户得到授权，但是访问是被禁止的。',
-  404: '发出的请求针对的是不存在的记录，服务器没有进行操作。',
-  405: '请求方法不被允许。',
-  406: '请求的格式不可得。',
-  410: '请求的资源被永久删除，且不会再得到的。',
-  422: '当创建一个对象时，发生一个验证错误。',
-  500: '服务器发生错误，请检查服务器。',
-  502: '网关错误。',
-  503: '服务不可用，服务器暂时过载或维护。',
-  504: '网关超时。',
-};
-
 /**
  * 异常处理程序
  */
 const errorHandler = (error: ResponseError) => {
+  const { messages } = getIntl(getLocale());
```

Comment:
I'm not sure if this is the right thing here -- could I possibly just use `getIntl()` instead?

Reply:
This also feels a bit cumbersome since I'm loading all the messages in here, but I struggled to figure out what else to use -- I was hoping I could use just `formatMessage` straight up, but it looked like I couldn't from what I could tell.